### DISCUSSION TITLE
Enhancement: User should be able to scroll through the list of the currently running download

### DISCUSSION THREAD

**Discussion by david-alleyn** [audiamus/BookLibConnect#15] [Ideas]
In my opinion, it would be a nice touch if the user could scroll through the list of the currently running download job.

![user-scroll](https://user-images.githubusercontent.com/185986/152321276-14c1bfeb-bb08-4ce6-bd41-31cd9308f42b.png)

I love both your apps. You've converted me over from OpenAudible, even more-so now that the Win10 app is gone.

**Comment by audiamus** [comment]
Yes, something like that.

One of the ideas is to automatically and temporarily move all books currently worked on to the top of the list. Theoretically, even the download can run in parallel (https://github.com/audiamus/BookLibConnect/issues/8#issuecomment-1023553625). 

After integration into AAX Audio Converter we will have the status line text messages again which also tells what's currently going on.

**Reply by audiamus** [reply]
Okay, this seems like becoming a 3 step enhancement. 

Step 1 is the easy one: enable scrolling. I do have to prevent any selection changes while the app is working. It would totally confuse the ongoing job. Until now I did this by simply disabling the list and the buttons. But this also affects the scroll bars. They disappear. However. with a bit of testing I have found a solution: I only need to intercept any mouse clicks in the list client area during the "run" phase and the selection will remain unchanged. Resorting must also be disabled during that time. But scrollbars will remain visible and workable.  

Step 2 will be the reshuffling of the list while working. But that will be postponed.

Step 3: After integration into AAX Audio Converter status text messages shall be added.
